tfcoreml/_tf_coreml_converter.py

```diff
@@ -642,30 +642,30 @@ def convert(tf_model_path,
       Custom shape function is required for adding custom layer in Core ML 3.
       If target_ios less than iOS 13 ('13'), then this option is ignored
 
-  target_ios: str
-      Target Deployment iOS Version (default: '12')
-      Supported iOS version options: '11.2', '12', '13'        
-      CoreML model produced by the converter will be compatible with the iOS version specified in this argument.
-      e.g. if target_ios = '12', the converter would only utilize CoreML features released till iOS12 (equivalently macOS 10.14, watchOS 5 etc).
+  min_target_ios: str
+      Minimum target deployment iOS version (default: '12')
+      Supported iOS version options: '11.2', '12', '13'
+      Core ML model produced by the converter will be compatible with the iOS version specified in this argument.
+      e.g. if min_target_ios='12', the converter would only utilize features released till iOS 12 (equivalently macOS 10.14, watchOS 5 etc).
```

Comment:
can we say here that `Core ML model produced by the converter will be compatible with the iOS version specified in this argument and the versions after it`. e.g.: if min_target_ios='12', the converter would only utilize layers released till iOS 12 (equivalently macOS 10.14, watchOS 5 etc), and the produced model can be deployed to iOS12+ (both iOS 12 and iOS 13)